src/server/routes/revision.js

```diff
@@ -437,162 +437,183 @@ function formatScalarChange(change, label) {
 	return [formatChange(change, label, (side) => side && [side])];
 }
 
-function formatEditionLanguages(change) {
+function formatNewReleaseEventSet(change) {
 	const rhs = change.rhs;
-
-	if (rhs && rhs.length) {
+	if (rhs.releaseEvents && rhs.releaseEvents.length > 0) {
 		return [formatRow(
-			'N', 'Language', null, rhs.map(
-				(language) => language.name
+			'N', 'Release Date', null, rhs.releaseEvents.map(
+				(releaseEvent) => releaseEvent.date
 			)
 		)];
 	}
 
-	return null;
+	return [];
 }
 
-function formatEditionLanguageAddOrDelete(change) {
+function formatReleaseEventAddOrDelete(change) {
+	const lhs = change.item.lhs && [change.item.lhs.date];
+	const rhs = change.item.rhs && [change.item.rhs.date];
+
 	return [
-		formatChange(
-			change.item,
-			`Language ${change.index}`,
-			(side) => side && [side]
-		)
+		formatRow(change.item.kind, 'Release Date', lhs, rhs)
```

Comment:
Could we use `formatChange()` here? Should be as simple as `formatChange(change.item, 'Release Date', (side) => side && [side.date])` if I'm not missing something.